src/Network/Network/FirewallPolicy/FirewallCustomRule/GroupByUserSession/GroupByVariable/NewAzureApplicationGatewayFirewallCustomRuleGroupByVariableCommand.cs

```diff
@@ -21,11 +22,12 @@ namespace Microsoft.Azure.Commands.Network
     [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApplicationGatewayFirewallCustomRuleGroupByVariable"), OutputType(typeof(PSApplicationGatewayFirewallCustomRuleGroupByVariable))]
     public class NewAzureApplicationGatewayFirewallCustomRuleGroupByVariableCommand : NetworkBaseCmdlet
     {
+        [CmdletParameterBreakingChangeWithVersionAttribute("VariableName", "11.0.0", "6.0.0", ChangeDescription = "Geo would be invalid for parameter VariableName")]
```

Comment:
The current Network version is 6.1.1. Thus, the 3rd parameter should be 7.0.0 (it means this parameter will be removed in 7.0.0).

Reply:
fixed